cli/utils/index.ts

```diff
@@ -43,6 +43,13 @@ export const assertShellResult = (result: ShellString, errMsg: string) => {
   }
 }
 
+export const assertFindings = (findings: Finding[]) => {
+  const serializedFindings = JSON.stringify(findings).length;
```

Comment:
from what i understood, each character in NodeJS is 2 bytes. i also found this method that is probably useful here: [`Buffer.byteLength'](https://www.w3schools.com/nodejs/met_buffer_bytelength.asp)

Reply:
Oh yep I think b/c UTF-8 encoding is the default for strings, will update the use Buffer.byteLength()

Reply:
Updated to use Buffer.byteLength